src/requests-and-replies.md

```diff
@@ -71,6 +74,9 @@ Multiple copies of each fragment may be sent to improve the chance of success. E
 fragment should contain different SURBs, and should be sent along a different route to the
 destination.
 
+// TODO is it possible to send without surb: just polling on chain if the transaction get eventually
+// included. I mean some users may not want to have this surb reply.
```

Comment:
For simplicity I've not bothered trying to support this.

Reply:
As a user it is certainly something that I would do (no reason to expose my query to a surb reply when I can just check publicly the node), but I can still send an invalid surb, it is just a bit of wasted space.
So my personal opinion is that query without surb should be supported (but maybe in another iteration).

Reply:
So I guess it is strictly speaking possible to send a request message without any SURBs. The existing node implementation would process the message and then simply fail to send a reply. I could say that this is how such messages should be handled in this spec. Then it would be up to nodes to support sending such messages or not (the existing implementation currently does not).